### PR TITLE
fix: add pure_pursuit as lateral controller into launch files

### PR DESCRIPTION
**Pull request by brkay54**
## Description

Closes  (#749)
Related (#567)
`tier4_control_launch/launch/control.launch.py` lateral_controller_mode option is not working.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_control_launch/launch/control.launch.py [hunk 1]
```diff
@@ -271,6 +297,12 @@ def add_launch_arg(name: str, default_value=None, description=None):
     #     "lateral controller mode: `mpc_follower` or `pure_pursuit`",
     # )
 
+    add_launch_arg(
```
Comment (maxime-clem) [inline launch/tier4_control_launch/launch/control.launch.py:300]: The same code is written in the previous lines but is commented out.

File: launch/tier4_control_launch/launch/control.launch.py [hunk 2]
```diff
@@ -325,6 +365,8 @@ def add_launch_arg(name: str, default_value=None, description=None):
     )
     add_launch_arg("enable_pub_debug", "true", "enable to publish debug information")
 
+    # lateral controller
+
```
Comment (maxime-clem) [inline launch/tier4_control_launch/launch/control.launch.py:369]: Should this be somewhere else ?

**Review by maxime-clem — COMMENTED** [review]
Tested and it works well.
I only have minor comments about some comments in the code.

**Review by maxime-clem — APPROVED** [review]

**Comment by brkay54** [comment]
@xmfcx Can you merge please?

**Pull request merged**